hyperspy/signal_tools.py

```diff
@@ -1003,12 +1005,15 @@ def set_background_estimator(self):
                 self.background_estimator = components1d.Polynomial(
                     self.polynomial_order)
             self.bg_line_range = 'full'
-        elif self.background_type == 'Lorentzian':
-            self.background_estimator = components1d.Lorentzian()
-            self.bg_line_range = 'full'
+        elif self.background_type == 'Power Law':
+            self.background_estimator = components1d.PowerLaw()
+            self.bg_line_range = 'from_left_range'
         elif self.background_type == 'SkewNormal':
             self.background_estimator = components1d.SkewNormal()
             self.bg_line_range = 'full'
+        elif self.background_type == 'Voigt':
+            with ignore_warning(message="The API of the `Voigt` component"):
+                self.background_estimator = components1d.Voigt()
```

Comment:
```suggestion
                self.background_estimator = components1d.Voigt(legacy=False)
```
to be consistent with the `remove_background` method.

Reply:
Thanks. I changed that.